subscriber/common/message/streamio_decoder.go

```diff
@@ -0,0 +1,105 @@
+package message
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/a8m/djson"
+	"github.com/uber/aresdb/subscriber/common/consumer"
+	"github.com/uber/aresdb/utils"
+	"go.uber.org/zap"
+)
+
+// StreamioDecoder is an implementation of Decoder interface for decoding the messages from Kafka
+// streamio topic
+type StreamioDecoder struct {
```

Comment:
shall we leave this in internal repo? I don't think anyone outside Uber would need this

Reply:
still leave it since it doesn't use any uber specific lib.